tutorials/quick_start/results/model_quantization_results.csv

```diff
@@ -1,13 +1,13 @@
 ModelName,ModelLibrary,DatasetName,TotalImages,FloatAcc,QuantAcc,Size[MB],BitsConfig,QuantTechnique,TPC
-mobilenet_v2,torchvision_lib,ImageNet,50000,0.7215,0.7188,3.47,W8A8,PTQ,default_pytorch_tpc-v4
-regnet_y_400mf,torchvision_lib,ImageNet,50000,0.7578,0.7542,4.31,W8A8,PTQ,default_pytorch_tpc-v4
-shufflenet_v2_x0_5,torchvision_lib,ImageNet,50000,0.6055,0.597,1.36,W8A8,PTQ,default_pytorch_tpc-v4
-squeezenet1_0,torchvision_lib,ImageNet,50000,0.581,0.5767,1.24,W8A8,PTQ,default_pytorch_tpc-v4
-regnetx_002,timm_lib,ImageNet,50000,0.6876,0.6827,2.67,W8A8,PTQ,default_pytorch_tpc-v4
-regnety_008,timm_lib,ImageNet,50000,0.7632,0.7598,6.23,W8A8,PTQ,default_pytorch_tpc-v4
-resnet10t,timm_lib,ImageNet,50000,0.6656,0.6643,5.43,W8A8,PTQ,default_pytorch_tpc-v4
-resnet18,timm_lib,ImageNet,50000,0.6976,0.6961,11.68,W8A8,PTQ,default_pytorch_tpc-v4
-efficientnet_es,timm_lib,ImageNet,50000,0.7808,0.7774,5.39,W8A8,PTQ,default_pytorch_tpc-v4
+mobilenet_v2,torchvision,ImageNet,50000,0.7215,0.7188,3.47,W8A8,PTQ,default_pytorch_tpc-v4
```

Comment:
The 'lib' suffix is meant to avoid conflicts with the original package names.
Please remove the folder name changes.

Reply:
The reason I did that is because it's inconsistent between code and example. And the code didn't work. It's obvious to me that this indicates where the model comes from, not the package name. There is no need for lib suffix even in the code because of the way the packages are loaded; no conflict possible. 
Either way, with or without lib suffix, is fine with me.

Reply:
@Idan-BenAmi 
Please take a look

Reply:
Hi,
Thanks for this comment! We already added a fix that resolved this naming problem here: https://github.com/sony/model_optimization/pull/785
Therefore this change is redundant now.